students/11th/hyojeung/login.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>instagram</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div class="warp">
+            <main class="mainBox">
+
+              <div class="logo">
+                <a href="#"><img alt="logo" src="img/logo_text.png"></a>
+              </div>
+
+              <div class="loginBox">
+              <div class="loginFom">
+                <input type = "text" id="idname" placeholder="전화번호, 사용자 이름 또는 이메일">
+                <input type = "password" id="password" placeholder="비밀번호"/>
+                <input type ="button" id="loginButton" value="로그인"/>
```

Comment:
```suggestion
              <div class="loginForm">
                <input type="text" id="idName" placeholder="전화번호, 사용자 이름 또는 이메일">
                <input type="password" id="password" placeholder="비밀번호" />
                <input type="button" id="loginButton" value="로그인" />
```
1. 속성값과 value 사이에 띄어쓰기 없애주세요
2. 변수명은 camel case로 작성해주세요.
3. 가능하면 id selector 대신 class selector 사용해주세요. id selector를 사용했을 때 우려될 수 있는 점을 댓글로 달아주세요.

Reply:
1. 사이의 띄어쓰기 수정완료 하였습니다.
2. 변수명 수정 전부 완료하였습니다.
3. id selector를 사용하면 반복문이 길어질수록 사용하기 어렵기때문에 우려되는것이...맞을까요...? 스크립트 부분이 짧아 id를 이용했었습니다, 수정하도록하겠습니다!